honeycombio/otel-launcher-go
# maint: add circleci config

## Which problem is this PR solving?

- Lack of CI

## Short description of the changes

- Add a Makefile for quick commands (currently has `test` to run `make test` from root directory and in CI)
- Add a CircleCI config to run tests and publish to Github

## How to verify that this has the expected result

- `make test` should pass tests
- nightly run and PR runs should successfully run in CircleCI
